github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -67,3 +67,32 @@ presubmits:
               memory: "1Gi"
             limits:
               memory: "1Gi"
+  - name: kubernetes-crud-ci-role
+    always_run: false
+    run_if_changed: "github/ci/kubernetes-crud/.*"
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-kubevirtci-docker-credential: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+        - image: gabrielecerami/kubevirt-ci-testenv
+          # docker run \
```

Comment:
Maybe you can move this into the README?

Reply:
You're right, but everything looks so ready that I don't want to ruin this moment.
I'll move the comment into README with the next patch.